article_store/app.cfg

```diff
@@ -0,0 +1,19 @@
+[django]
+ci = false
+debug = true
+allowed_hosts = web,article-store,localhost,127.0.0.1
```

Comment:
`web` is?

Reply:
...the old name for the container and no longer needed 👍 will remove